recipe/meta.yaml

```diff
@@ -33,11 +20,10 @@ requirements:
     - python
     - pip
     - cython
-    - proj4
+    - proj4 >=6.0.*
```

Comment:
We should update our global pinning to latest proj.

PS: I'm guessing that `pyproj` still works with `proj.4 5.*`, right? If not then we do have to require it here too.

Reply:
No, it doesn't work with older versions of Proj.4

Reply:
I'm not sure about the ABI but I guess we have to do something like:

```yaml
build:
  - proj4 6.0.*
run:
  - proj4 >=6.0
```

Imagining an ABI compatibility with micro version like `x.x`.